app/validators/subnets_consistency_validator.rb

```diff
@@ -0,0 +1,22 @@
+class SubnetsConsistencyValidator < ActiveModel::Validator
+  def validate(record)
+    return unless record.subnet && record.subnet6
+    validate_mtu(record)
+    validate_vlanid(record)
+  end
+
+  def validate_mtu(record)
+    return unless record.subnet.mtu != record.subnet6.mtu
+    add_error(record, _('MTU is not consistent accross subnets'))
+  end
+
+  def validate_vlanid(record)
+    return unless record.subnet.vlanid != record.subnet6.vlanid
```

Comment:
`Vlanid` is currently not required. Should we ignore this if it's any one or both of the subnets?

Reply:
This will be fine if both subnets don't have vlanid. IMHO having subnets with different vlanid (different values or one tagged and the other one not), is a nonsense.